config/manifests/metallb-frr-webhooks.yaml

```diff
@@ -0,0 +1,1662 @@
+apiVersion: v1
```

Comment:
what's the difference between this file and `metallb-frr-with-webhooks`? same for the native pair

Reply:
One is legacy, it's not getting generated. Removing it.